### DISCUSSION TITLE
Unable to run pybamm

### DISCUSSION THREAD

**Discussion by amirsh213** [pybamm-team/PyBaMM#1896] [Q&A]
Hi, 
I'm trying to use pybamm with git bash (Windows) to run %%bash commands. I activated conda and ran bash commands (in Jupyter) but then I cannot run pybamm. I already used pybamm (w/o git bash) without any problem but I don't know where this error comes from:
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_9224/62250946.py in <module>
      3 model=pybamm.lithium_ion.DFN()
      4 
----> 5 sim=pybamm.Simulation(model)
      6 
      7 sim.solve([0, 3600])

~\Anaconda\PyBaMM-21.08\pybamm\simulation.py in __init__(self, model, experiment, geometry, parameter_values, submesh_types, var_pts, spatial_methods, solver, output_variables, C_rate)
     83         C_rate=None,
     84     ):
---> 85         self.parameter_values = parameter_values or model.default_parameter_values
     86 
     87         if isinstance(model, pybamm.lithium_ion.BasicDFNHalfCell):

~\Anaconda\PyBaMM-21.08\pybamm\models\full_battery_models\base_battery_model.py in default_parameter_values(self)
    405         # Default parameter values
    406         # Lion parameters left as default parameter set for tests
--> 407         return pybamm.ParameterValues(chemistry=pybamm.parameter_sets.Marquis2019)
    408 
    409     @property

~\Anaconda\PyBaMM-21.08\pybamm\parameters\parameter_values.py in __init__(self, values, chemistry)
     72         # First load chemistry
     73         if chemistry is not None:
---> 74             self.update_from_chemistry(chemistry)
     75         # Then update with values dictionary or file
     76         if values is not None:

~\Anaconda\PyBaMM-21.08\pybamm\parameters\parameter_values.py in update_from_chemistry(self, chemistry)
    215 
    216             # Update parameters, making sure to check any conflicts
--> 217             self.update(
    218                 component_params,
    219                 check_conflict=True,

~\Anaconda\PyBaMM-21.08\pybamm\parameters\parameter_values.py in update(self, values, check_conflict, check_already_exists, path)
    303             if isinstance(value, str):
    304                 if value.startswith("[function]"):
--> 305                     loaded_value = pybamm.load_function(os.path.join(path, value[10:]))
    306                     self._dict_items[name] = loaded_value
    307                     values[name] = loaded_value

~\Anaconda\PyBaMM-21.08\pybamm\util.py in load_function(filename)
    271     path = root_path.replace("/", ".")
    272     path = path.replace("\\", ".")
--> 273     module_object = importlib.import_module(path)
    274 
    275     return getattr(module_object, tail)

~\anaconda3\lib\importlib\__init__.py in import_module(name, package)
    125                 break
    126             level += 1
--> 127     return _bootstrap._gcd_import(name[level:], package, level)
    128 
    129 

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _call_with_frames_removed(f, *args, **kwds)

~\anaconda3\lib\importlib\_bootstrap.py in _gcd_import(name, package, level)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load(name, import_)

~\anaconda3\lib\importlib\_bootstrap.py in _find_and_load_unlocked(name, import_)

ModuleNotFoundError: No module named 'C:'

**Comment by Saransh-cpp** [comment]
@amirsh213 can you share your script? This was also reported here - #1804. It should be fixed by the next release.
